platform/feature-api/api/pod-api/src/spec.ts

```diff
@@ -62,7 +62,7 @@ export class PodSpec {
                 await assert.isRejected(polyIn.delete(quad), /default/);
             });
 
-            it("add/select", async () => {
+            it("add/match", async () => {
```

Comment:
I already changed that in #814. Anyway, I did exactly the same, so no big deal.

Reply:
Ah, thought it had been forgotten, since the code is already using `match` :D Let's merge your changes first then!